depends/README.md

```diff
@@ -90,9 +86,13 @@ For linux S390X cross compilation:
 
     pkg install bash
 
+### Install the required dependencies: NetBSD
+
+    pkgin install bash gmake
+
 ### Install the required dependencies: OpenBSD
 
-    pkg_add bash gtar
+    pkg_add bash gmake gtar
```

Comment:
None of these package are required for the native build with CMake.

Reply:
> Otherwise, `autoconf automake` should be listed here as well, because they are still required for some `depends` packages? In any case you are removing `autoconf automake` and `gmake` from the instructions in `doc/build-openbsd.md`, so overall it seems a bit inconsistent?

I agree that this issue has to be resolved in `depends/README.md`.

Reply:
> Otherwise, `autoconf automake` should be listed here as well, because they are still required for some `depends` packages?

Apparently, they are not. Thanks to the recent migration of dependency packages to CMake.

cc @theStack

Reply:
> Apparently, they are not. 

SQLite still uses Autotools?

Reply:
> Just to clarify, removing `gmake` from the instructions in `doc/build-openbsd.md` is also intentional in this pull request?

Yes, it is. CMake can work with any flavour of Make, so GNU Make is not longer a requirement.

Reply:
> > Apparently, they are not.
> 
> SQLite still uses Autotools?

The `sqlite-autoconf-3380500.tar.gz` archive contains pre-built `configure` script and `Makefile.in` file. Therefore, Autotools is not required on our side.

Reply:
Autotools might not be used currently for sqlite (libtool still is), but we can't rule out entirely that it wont be used going forward. For example, if we need to regenerate the configure script for some reason. This has happened in the past, for example, something like 6897c4bdf51a4aa74320ebfffa9467db14670109. This might be very unlikely here, but not impossible. 

Note that this would also breakdown, if we were to shift to building from source (requiring us to generate the build scripts oursevles), rather than using the tarballs with everything pre-generated (and unverifiable/opauge).

Reply:
Right. Such a possibility does exist. However, instructing the user to install packages that are not currently required does not seem compelling to me.

Reply:
> Autotools might not be used currently for sqlite (libtool still is)...

The tarball also contains a pre-built `ltmain.sh` script, so `libtool` package installation is not needed.

Reply:
This PR does not change how packages are built in depends. Therefore, GNU Make is still a requirement to build depends.

Reply:
> But the document still refers to `gmake`, no? For example line 47 of `doc/build-openbsd.md` says:
> 
> ```
> gmake -C depends NO_BOOST=1 NO_LIBEVENT=1 NO_QT=1 NO_SQLITE=1 NO_NATPMP=1 NO_UPNP=1 NO_ZMQ=1 NO_USDT=1
> ```

The line above refers to `depends/README.md`, where the required build dependencies, including `gmake`, are suggested to be installed.

Reply:
Resolving this conversation based "thumbs up" emoji.